idearium-lib/lib/query.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+/**
+ * Assign some defaults to the passed Mongoose query options.
+ * @param {Object} options Mongoose query options.
+ * @return {Object} Returns the options object for Mongoose queries.
+ */
+const getOptions = (options) => {
+
+    const defaults = {
+        filter: {},
+        lean: true,
+        limit: 10,
```

Comment:
I've added a limit since this file was last used, I think 10 is sufficient as a default?